app/main.py

```diff
@@ -2,0 +1,12 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
+    if command != "mv":
```

Comment:
Code Efficiency: You are checking the command after splitting it. It would be more efficient to check if the command starts with 'mv' before splitting it.